docs/content/get-started/developers.mdx

```diff
@@ -5,18 +5,190 @@ nav_order: 20
-Nullam sit amet turpis vel nisi tristique dapibus sed in libero.
-Vivamus ac tristique nisl, eu bibendum eros.
+1. [Phase 1: Install](#phase-1-install)
+2. [Phase 2: Compile](#phase-2-compile)
+3. [Phase 3: Customize](#phase-3-customize)
 
-## Our Goals
+## Phase 1: Install
 
-Aenean semper sit amet quam sit amet convallis. Nulla facilisi. Fusce lacinia arcu molestie,
-pharetra mauris pharetra, tristique leo. Ut neque diam, malesuada dictum faucibus vitae,
-blandit a lacus.
+The NCI Design System distributes our source code through npm, a package manager that uses a flavor of JavaScript called Node.js. The best way to add the Design System to your project is with npm via a Terminal window (check out some tips for using Terminal on a [Mac](https://support.apple.com/guide/terminal/open-or-quit-terminal-apd5265185d-f365-44cb-8b09-71a064a42125/mac) or [Windows](https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701?rtc=1&activetab=pivot:overviewtab)).
+
+Installing the Design System with Node and npm not only allows you to install all the code the Design System needs to compile with just a couple commands, but will version any installed packages, like NCIDS, as well — meaning your project code is tied to a specific version of the Design System. Confirming what version you’re using and updating to a newer (or older) version is straightforward.
+
+### Step 1: Install Node and npm
+
+Ensure that you have Node.js installed, preferably Node 18, to meet the required runtime environment.
+
+Open your Terminal application and a Terminal window. Check to see if you have Node installed with `node -v`.
+
+If you don’t have Node, we recommend installing it through a node version manager such as [NVM](https://github.com/nvm-sh/nvm) on a Mac or Linux machine, or [nvm-windows](https://github.com/coreybutler/nvm-windows) on a Windows machine. It can also be installed directly from [Node.js](https://nodejs.org/en/download/).
+
+### Step 2: Authenticating to the NCIDS npm registry
+
+NCIDS utilizes GitHub’s npm registry to host its packages. To access and download these packages, you need to be authenticated with a valid access token.
+
+1. Create your personal access token
+   Follow the steps outlined in GitHub’s documentation for [creating a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+   Ensure that, at a minimum, the required scopes are set to “repo,” “read:org,” and “workflow.”
+
+2. Authenticate with GitHub
+   Run the following command to authenticate with GitHub using the GitHub CLI:
+   `gh auth login`
+
+3. Log in to the @NCIOCPL Organizational Scope on npm
+   Run the following command to log in to npm with the organizational scope and using the GitHub npm registry:
+   `npm login --scope=@NCIOCPL --registry=https://npm.pkg.github.com`
+
+**_Note_** You must use the personal access token generated in step 1 as your password.
```

Comment:
This `note` is missing a colon and styling does not match the note found on line 184